### PR TITLE
Refactor to use mbtiles and backend from jeo-core

### PR DESCRIPTION
**Pull request by jericks**
Solves issue #7.

Thanks!
Jared

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,16 @@
       <artifactId>jeo-sql</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
```
Comment (jdeolive) [inline pom.xml]: indentation a bit off.

File: src/test/java/org/jeo/android/geopkg/GeoPkgWorkspaceTest.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
  */
 package org.jeo.android.geopkg;
 
+import org.jeo.android.SQLiteBackend;
```
Comment (jdeolive) [inline src/test/java/org/jeo/android/geopkg/GeoPkgWorkspaceTest.java]: superfluous import?

**Comment by jdeolive** [comment]
A few nit-picky trivial things but it looks awesome. I believe you have rights to push so go ahead and push at your discretion.

**Comment by jericks** [comment]
Hey Justin!

Sorry about the formatting, it's fixed now.  The import is necessary because it was moved from the org.jeo.android.geopkg package to the org.jeo.android package.

Jared

**Pull request merged**